build.sbt

```diff
@@ -1,24 +1,30 @@
-val commonScalacOptions =
-  Seq(
-    "-feature",
-    "-deprecation",
-    "-Xlint",
-    "-Xfatal-warnings"
-  )
-
 val Scala212 = "2.12.19"
 val Scala213 = "2.13.13"
+val Scala3 = "3.3.3"
```

Comment:
Perhaps it's the most controversial thing. The recent version of Scala 3 is `3.4.2`. But since Scala 3 doesn't keep forward compatibility between minor versions, we should barely stick with it. Also, Scala 3.3 is an LTS — https://virtuslab.com/blog/technology/scala-3-roadmap-for-2024/#at-least-one-lts-patch-for-every-two-next-patch-releases. Last but not least, many libraries I used to contribute have decided to stick with 3.3. I propose to act similarly.